$NetBSD$

# HG changeset patch
# User Tomas Mraz <tmraz@redhat.com>
# Date 1458546851 -3600
# Node ID 3447d64e5ab82e9469f21371c540b09456874262
# Parent  a4696cd7cd246f6c34a2d2f1d54d569c5e7ecd84
Add missing parts to pwquality.pc.in

--- src/pwquality.pc.in.orig	2012-12-19 16:52:09.000000000 +0000
+++ src/pwquality.pc.in
@@ -1,5 +1,11 @@
+prefix=@prefix@
+exec_prefix=@exec_prefix@
+libdir=@libdir@
+includedir=@includedir@
+
 Name: pwquality
 Description: Password quality checker
 Version: @PACKAGE_VERSION@
-Libs: -lpwquality
+Libs: -L${libdir} -lpwquality
 Libs.private: -lcrack
+Cflags: -I${includedir}
